phpstan.neon.dist

```diff
@@ -1,13 +1,18 @@
-parameters:
-    paths:
-        - %currentWorkingDirectory%/
-    excludes_analyse:
-          - %currentWorkingDirectory%/vendor/php-stubs/wordpress-stubs/wordpress-stubs.php 
-    autoload_files:
-#        # Procedural code
-         - %currentWorkingDirectory%/util.php
-         #- %currentWorkingDirectory%/altapay.php
-         #stubs
-         - %currentWorkingDirectory%/stubs/wc_get_product.php
-    autoload_directories:
-         - %currentWorkingDirectory%/
+parameters:
+    paths:
+        - %currentWorkingDirectory%/
+    excludes_analyse:
+        - %currentWorkingDirectory%/vendor/
+        - %currentWorkingDirectory%/assets/
+    scanDirectories:
+        - ../woocommerce/
+        - ../../../wp-includes/
+        - ../../../wp-admin/
+#        - ../woocommerce-subscriptions
+#        - ../wordpress-develop-tests
+    level: 0
+    checkMissingIterableValueType: false
+    ignoreErrors:
+        - "#Inner named functions are not supported by PHPStan#"
+        - "#Function wcs_order_contains_renewal not found.#"
+        - "#unknown class WC_Subscriptions_Renewal_Order#"

```

Comment:
These appear to come from the https://github.com/wp-premium/woocommerce-subscriptions WC-plugin. We should install it and probably also as a minimum mention our integration with it in the readme.